packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js

```diff
@@ -96,20 +104,20 @@ const CheckBoxView = ({
         { tooltip ? (
           <Tooltip id={datafield} title={tooltip}>
             <div className={datafield}>
-              <LabelComponent />
+              {customName || name}
             </div>
           </Tooltip>
         ) : (
           <LabelComponent />
         )}
-        <ListItemText />
+        <ListItemText className={`${checkedSection}_md_space`} />
         <Typography
           className={clsx(`${checkedSection}Subjects`, {
             [`${checkedSection}SubjectUnChecked`]: !isChecked,
             [`${checkedSection}SubjectChecked`]: isChecked,
           })}
         >
-          {`(${subjects})`}
+          {customSubjects || `(${subjects})`}
```

Comment:
Do we need our documentation updated as we have added customSubjects and customName?